ui-src/README.md

```diff
@@ -0,0 +1,27 @@
+# Angular
```

Comment:
Can we update the main README with these commands or the commands that your using to get this code running. Then get rid of this one.